packages/jwt-verifier/package.json

```diff
@@ -7,14 +7,15 @@
   "main": "lib.js",
   "config": {
     "tck": {
-      "jar": "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=com.okta.oidc.tck&a=okta-oidc-tck&v=0.2.1-SNAPSHOT&e=jar&c=shaded",
-      "setup_env": "cross-env ISSUER=http://localhost:9090/oauth2/default cross-env CLIENT_ID=OOICU812 cross-env CLIENT_SECRET=VERY_SECRET",
-      "command": "java -Dconfig=./test/integration-test/resources/testRunner.yml -cp ./okta-oidc-tck-0.2.1-SNAPSHOT-shaded.jar org.testng.TestNG -d target/cli-test-output ./test/integration-test/resources/testng.xml"
+      "jar": "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=com.okta.oidc.tck&a=okta-oidc-tck&v=0.2.2-SNAPSHOT&e=jar&c=shaded",
+      "command": "java -Dconfig=./test/integration-test/resources/testRunner.yml -cp ./okta-oidc-tck-0.2.2-SNAPSHOT-shaded.jar org.testng.TestNG -d target/cli-test-output ./test/integration-test/resources/testng.xml"
```

Comment:
We should try to make `okta-oidc-tck-*` a variable, that way we only need to replace this once when we update the version

Reply:
Moving this to a separate script which would solve this problem